src/condor_ce_router_defaults

```diff
@@ -50,15 +50,23 @@ JOB_ROUTER_DEFAULTS_GENERATED = \\
     set_requirements = True; \\
     /* Note default memory request of 2GB */ \\
     /* Note yet another nested condition allow pass attributes (maxMemory,xcount,jobtype,queue) via gWMS Factory described within ClassAd */ \\
-    eval_set_RequestMemory = ifThenElse(maxMemory isnt undefined, maxMemory, ifThenElse(default_maxMemory isnt undefined, default_maxMemory, 2000)); \\
+    eval_set_OriginalMemory = ifThenElse(maxMemory isnt undefined, maxMemory, ifThenElse(default_maxMemory isnt undefined, default_maxMemory, 2000)); \\
+    set_JOB_GLIDEIN_Memory = "$$(TotalMemory:0)"; \\
+    set_JobMemory = JobIsRunning ? int(MATCH_EXP_JOB_GLIDEIN_Memory)*95/100 : OriginalMemory; \\
```

Comment:
The binary ternary is new in 8.6. The ternary operator has been there since 1999.